fhir-server-test/src/test/java/com/ibm/fhir/server/test/R4ExampleServerTest.java

```diff
@@ -16,21 +16,21 @@
  * Basic sniff test of the FHIR Server.
  */
 public class R4ExampleServerTest extends FHIRServerTestBase {
-    
+
     /**
      * Process all the examples in the fhir-r4-spec example library
      */
     @Test(groups = { "server-examples" })
     public void processExamples() throws Exception {
         DriverMetrics dm = new DriverMetrics();
-        
+
         // Process each of the examples using the provided ExampleRequestProcessor. We want to
         // validate first before we try and send to FHIR
         final R4ExamplesDriver driver = new R4ExamplesDriver();
         driver.setMetrics(dm);
         driver.setValidator(new ValidationProcessor());
-        driver.setProcessor(new ExampleRequestProcessor(this, null, dm, 1));
-        
+        driver.setProcessor(new ExampleRequestProcessor(this, "default", dm, 1));
```

Comment:
remind me again what this is for and why we need to pass default now, when null worked before?

Reply:
null didn't work for db2 test, I believe we had never test it on db2 before with null (at least after Robin's change) for db2 multi-tenant support, and I believe that's caused by that currently we need a db2 tenant key for  for each tenant in the database(s), and there is no tenant key for a null tenant.  the mapping is done in Robin's tenant management table in each database...